src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -43,6 +43,20 @@ public MilestonePickerDialog(Stage stage, TurboIssue issue, List<TurboMilestone>
         state = new MilestonePickerState(originalMilestones);
         initUI();
         setupKeyEvents();
+        fillInputField();
+    }
+
+    /**
+     * Fills the input field of milestone picker dialog
+     * If there is an existing milestone, input field is filled with existing milestone's title
+     * Else input field is filled with the next open milestone that is not overdue
+     */
+    private void fillInputField() {
```

Comment:
Maybe `setInputFieldToDefaultMilestone` or something similar would be more explanative. Define default milestone (or other better terms you can come up with) on header comment.